.travis.yml

```diff
@@ -3,7 +3,7 @@ notifications:
 language: ruby
 script:
 - bundle exec rubocop
-- cd tests && ./test.sh && cd ..
+- (tests/test.sh)
```

Comment:
Yeah, I'm not sure how it could unless somebody adds `cd` back in here. The dir change inside the script won't propagate out.